src/main/java/mate/academy/hibernate/relations/dao/impl/CountryDaoImpl.java

```diff
@@ -12,11 +15,39 @@ public CountryDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Country add(Country country) {
-        return null;
+        Session session = null;
```

Comment:
You don't need to declare the session and transaction variables outside of the try block. This is a code style issue. You can declare them inside the try block to limit their scope.